openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/diagnostic/DiagnosticMessages.java

```diff
@@ -55,7 +55,13 @@ public enum DiagnosticMessages {
             DiagnosticSeverity.WARNING),
     OAS_CLIENT_114("OAS_CLIENT_114", "the operation: '%s' is skipped in the client generation since only a default" +
             " response found for the operation which is not supported with status code binding option",
-            DiagnosticSeverity.WARNING);
+            DiagnosticSeverity.WARNING),
+    OAS_CLIENT_115("OAS_CLIENT_115", "the 'default' response for the operation: '%s' is skipped in the client " +
```

Comment:
What is this error? We support mapping default status code responses right?

Reply:
This can be the conflict with the previous `default` status code changes , so we can remove this message